contracts/vaults/Claim.sol

```diff
@@ -93,29 +102,34 @@ contract Claim is Base {
         lpToken.safeTransfer(submittedClaim.beneficiary, claimBounty.hacker);
         lpToken.safeTransfer(submittedClaim.committee, claimBounty.committee);
         //storing the amount of token which can be swap and burned so it could be swapAndBurn in a separate tx.
-        swapAndBurns[_pid] += claimBounty.swapAndBurn;
-        governanceHatRewards[_pid] += claimBounty.governanceHat;
-        hackersHatRewards[submittedClaim.beneficiary][_pid] += claimBounty.hackerHatVested;
-
-        emit ApproveClaim(_pid,
-                        msg.sender,
-                        submittedClaim.beneficiary,
-                        submittedClaim.bountyPercentage,
-                        tokenLock,
-                        claimBounty);
-        assert(poolInfos[_pid].balance > 0);
+        swapAndBurns[pid] += claimBounty.swapAndBurn;
+        governanceHatRewards[pid] += claimBounty.governanceHat;
+        hackersHatRewards[submittedClaim.beneficiary][pid] += claimBounty.hackerHatVested;
+        delete activeClaims[pid];
+        delete submittedClaims[_claimId];
```

Comment:
is there a reason to keep this maybe? and check activeClaims in approve and dismiss?

Reply:
Well, it saves some gas. And the data will remain available to offchain apps bc we have the events, so there is no real need to keep it